src/gmp/gmp.ts

```diff
@@ -89,6 +91,8 @@ class Gmp {
   readonly agentinstaller: AgentInstallerCommand;
   readonly agentinstallers: AgentInstallersCommand;
   readonly auth: AuthenticationCommand;
+  readonly credentialStore: CredentialStoreCommand;
+  readonly credentialStores: CredentialStoresCommand;
```

Comment:
The current code expects the commands to be of type EntityType and therefore to be all lower case. I am really not sure about possible consequences if we change that now.
```suggestion
  readonly credentialstore: CredentialStoreCommand;
  readonly credentialstores: CredentialStoresCommand;
```